test/unit/controllers/ChoiceControllerSpec.scala

```diff
@@ -18,9 +18,10 @@ package unit.controllers
 
 import controllers.consolidations.{routes => consolidationRoutes}
 import controllers.{routes, ChoiceController}
-import forms.Choice
+import forms.{AssociateKind, AssociateUcr, Choice, ConsignmentReferences, DisassociateKind, DisassociateUcr, ShutMucr, UcrType}
```

Comment:
Nit: in such cases it is safer to use `forms._` because sometimes code formatters are complaining about the order.